sdk/arm/resources/2020-06-01/armresources/zz_generated_models.go

```diff
@@ -1540,69 +1398,22 @@ type Resource struct {
 	Name *string `json:"name,omitempty" azure:"ro"`
 
 	// Resource tags
-	Tags *map[string]string `json:"tags,omitempty"`
+	Tags *map[string]*string `json:"tags,omitempty"`
 
 	// READ-ONLY; Resource type
 	Type *string `json:"type,omitempty" azure:"ro"`
 }
 
 func (r Resource) marshalInternal() map[string]interface{} {
 	objectMap := make(map[string]interface{})
-	if r.ID != nil {
-		objectMap["id"] = r.ID
-	}
-	if r.Location != nil {
-		objectMap["location"] = r.Location
-	}
-	if r.Name != nil {
-		objectMap["name"] = r.Name
-	}
-	if r.Tags != nil {
-		objectMap["tags"] = r.Tags
-	}
-	if r.Type != nil {
-		objectMap["type"] = r.Type
-	}
+	populate(objectMap, "id", r.ID)
+	populate(objectMap, "location", r.Location)
+	populate(objectMap, "name", r.Name)
+	populate(objectMap, "tags", r.Tags)
+	populate(objectMap, "type", r.Type)
 	return objectMap
 }
 
-func (r *Resource) unmarshalInternal(rawMsg map[string]*json.RawMessage) error {
```

Comment:
I am not sure why this was deleted here, but this caused issues in the new version of this module
Could you take a look? @jhendrixMSFT

Reply:
I have a fix for this https://github.com/Azure/autorest.go/pull/611

Can you add this back in the meantime?

Reply:
Because we are tracking the generation metadata in this module too, I prefer to wait for a new release of autorest.go and then regenerate to fix this one.

Reply:
The `preview.19` generator will release by the time you read this with the fix.  Can you please regenerate all track 2 PRs with this version?